build/Containerfile

```diff
@@ -10,7 +10,7 @@ COPY . /src
 WORKDIR /src
 RUN mkdir /out
 
-RUN task_names=("copy-template" "create-vm" "execute-in-vm" "generate-ssh-keys" "modify-data-object" "modify-vm-template" "wait-for-vmi-status"); \
+RUN task_names=("copy-template" "create-vm" "execute-in-vm" "generate-ssh-keys" "modify-data-object" "modify-vm-template" "wait-for-vmi-status" "disk-uploader"); \
```

Comment:
this change should go to this file https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/build/Containerfile.DiskVirt#L14